R/run_prep_calculation_loans.R

```diff
@@ -185,7 +176,8 @@ run_prep_calculation_loans <- function(data_path_project_specific,
       .data$comp_loan_size_credit_limit,
       .data$loan_size_credit_limit_currency
     ) %>%
-    # TODO why distinct_all?
+    # ADO 2393 - somehow the summarise call does not reduce the number of rows.
```

Comment:
that is a bit of a scary comment. If your expectation was that summarise should reduced rows you might want to sharply look at it once more, maybe there are some unexpected patterns in the data set

Reply:
found the issue, weird aggregation.. came to the same result, but now it is much more concise